src/stats/inc/TKGroup.h

```diff
@@ -115,6 +115,14 @@ class BaseTKGroup {
    * After this method is called, covMatrixIsDirty should return \c true.
    */
   virtual void cleanCovMatrix() = 0;
+
+  //! Sets and proposal covariance matrix (and for all DR stage RVs as well)
+  /*!
+   * If the TK has a proposal covariance matrix, set the covariance matrix.
+   * If the TK support delayed rejection, derived classes should scale by a
+   * factor of \f$ 1/scales^2 \f$.
+   */
+  virtual void updateLawCovMatrix(const M & covMatrix) = 0;
```

Comment:
Note, this change break backwards compatibility.  In my defence, I highly doubt anybody is subclassing TKs yet; the factory infrastructure is quite new.

Instead, I can make this function virtual but non-pure and throw a run-time exception instead.